timer-8018x.c

```diff
@@ -5,7 +5,199 @@
 #include "emu-int.h"
 #include "emu-timer.h"
 
+#include "emu-int.h"
```

Comment:
Header `emu-int.h` included twice ?

Reply:
must be copy paste problem.

The code is not 100percent merge ready, coding style is not 100percent same as rest of the project. I would say that this pull request is "preview" for now. Ill fix them as time permits, and if there is interest from community :)

Reply:
Will fix it later